Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableAssetsCache.cs

```diff
@@ -1,4 +1,7 @@
-﻿using System;
+﻿using DCL.PerformanceAndDiagnostics.Optimization.PerformanceBudgeting;
+using DCL.PerformanceAndDiagnostics.Optimization.Priority_Queue;
```

Comment:
Can we adjust namespaces? I'd also propose to shorten them. Maybe skip "PerformanceDiagnostics"?